Makefile

```diff
@@ -10,7 +10,7 @@ all: bin/tornjak-agent bin/tornjak-manager ui-agent ui-manager container-agent c
 
 bin/tornjak-agent: $(GO_FILES) vendor
 	# Build hack because of flake of imported go module
-	docker run --rm -v "${PWD}":/usr/src/myapp -w /usr/src/myapp -e GOOS=linux -e GOARCH=amd64 golang:1.16 /bin/sh -c "go build --tags 'sqlite_json' tornjak-backend/cmd/agent/agent.go; go build --tags 'sqlite_json' -mod=vendor -ldflags '-s -w -linkmode external -extldflags "-static"' -o bin/tornjak-agent tornjak-backend/cmd/agent/agent.go"
+	docker run --rm -v "${PWD}":/usr/src/myapp -w /usr/src/myapp -e GOOS=linux -e GOARCH=arm64 golang:1.16 /bin/sh -c "go build --tags 'sqlite_json' tornjak-backend/cmd/agent/agent.go; go build --tags 'sqlite_json' -mod=vendor -ldflags '-s -w -linkmode external -extldflags "-static"' -o bin/tornjak-agent tornjak-backend/cmd/agent/agent.go"
```

Comment:
let's change this back for now
in general, we will likely not change the Makefile, so no need to add the Makefile before committing! 
i think this will fix the sqlite3 errors that are coming up in that other file. 
let's also try to build locally with the former GOARCH and see if it works

Reply:
also I can't comment on package-lock.json but let's revert back to the original package-lock file on main because we are not changing the frontend currently. we do need to fix this in a later pr tho!